src/Elastic.Markdown/Myst/InlineParsers/DiagnosticLinkInlineParser.cs

```diff
@@ -151,7 +151,7 @@ public override bool Match(InlineProcessor processor, ref StringSlice slice)
 		// rooted links might need the configured path prefix to properly link
 		var prefix = processor.GetBuildContext().UrlPathPrefix;
 		if (url.StartsWith("/") && !string.IsNullOrWhiteSpace(prefix))
-			link.Url = $"{prefix}/{link.Url}";
+			link.Url = $"{prefix}{link.Url}";
```

Comment:
```suggestion
			link.Url = $"{prefix.TrimEnd('/')}{link.Url}";
```

That way users can pass `--path-prefix wihout/a/slash` and `--path-prefix with/a/slash/`

Reply:
I see. Thank you!